pyodide_build/buildall.py

```diff
@@ -129,6 +129,8 @@ def generate_dependency_graph(
 
     while packages:
         pkgname = packages.pop()
+        if not pkgname:
+            continue
```

Comment:
I think we should rather get rid of those in `_parse_package_subset`. Here `pkgname` is str, so the fact that it can be an empty string is worrisome (or otherwise it means that type annotations are not correct)

Reply:
It's filtering empty string. If `PYODIDE_PACKAGES="micropip, " then `packages=["micropip", ""]`.

Reply:
Updated.